api/models/todoListModels.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const TodoSchema = new Schema({
+  name: {
+    type: String,
+    Required: 'Todo needs a name',
```

Comment:
if we use the validation strategy above, you can get rid of this required message.